core/src/main/java/org/elasticsearch/action/get/MultiGetResponse.java

```diff
@@ -149,6 +166,65 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         return builder;
     }
 
+    public static MultiGetResponse fromXContent(XContentParser parser) throws IOException {
+        String currentFieldName = null;
+        List<MultiGetItemResponse> items = new ArrayList<>();
+        for (Token token = parser.nextToken(); token != Token.END_OBJECT; token = parser.nextToken()) {
+            switch (token) {
```

Comment:
Does this switch have a default case?

Reply:
No, there is no default case. Normally I would add a default case that throws an exception, but we should be lenient here.

Reply:
I see, can you still add an empty default case, maybe with a comment that we ignore those cases? I get a warning here that could be avoided.